### ISSUE TITLE
Support: ZeroMQ installation/ AWS MQ usage

### ISSUE THREAD

**Issue by lu-chi** [MISP/MISP#8379]
### Support Questions

Hello there,

I am wondering if it's possible use AWS MQ rather then ZeroMQ installation. Does anyone of you tried that at some point? What would be necessary as a CLI on an instance, how this works, what was your general experience/feeling about that?

### MISP version

2.4.158

### Operating System

Ubuntu

### Operating System version

18

### PHP version

7.4

### Browser

all

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by adulau** [comment]
Indeed, it could be possible to extend MISP for AWS MQ.  The code base already including ZMQ and Kafka which are basically using the same approaches. 

I see three potential options:

- Extending MISP to support it. We don't have any user in our communities requiring AWS MQ. If you really need this, you can have a look at our professional services https://www.misp-project.org/professional-services/ and we could do a proposal if needed.
- Proposing a pull-request. I would recommend to have a look at the current part of the code dealing with ZMQ/Kafka it would be the exact same approach. Then we would review it and merge the contribution.
- We are currently working on a new type of misp modules to trigger specific action on publishing (for the next version). Extending misp-modules would be possible to do such kind of thing.

**Comment by lu-chi** [comment]
Thank you for providing info on that and apologies for very late reply - I really appreciate it. I guess I will wait for a new release/version. In a meantime I will continue experimenting with splitting MISP extensions across/with AWS specific services.
